src/components/MapView/PendingMapView.tsx

```diff
@@ -23,8 +23,8 @@ function PendingMapView({title = '', subtitle = ''}: PendingMapViewProps) {
                 <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter, styles.ph10]}>
                     <Icon
                         src={Expensicons.EmptyStateRoutePending}
-                        width={100}
-                        height={100}
+                        width={120}
```

Comment:
NAB: Shouldn't this be a style variable?

Reply:
I think it should, that's why I asked here:

> What are the precise dimensions of the icon in the new design? And what should I call the new size? As it looks larger than the current max: iconSizeSuperLarge: 60

I will name it `iconSizeUltraLarge`, but in general I think this naming needs some standartization.